agent/agent.go

```diff
@@ -1575,8 +1578,16 @@ func (a *agent) manageProcessPriorityUntilGracefulShutdown() {
 		a.processManagementTick = ticker.C
 	}
 
+	oomScore := unsetOOMScore
+	if scoreStr, ok := a.environmentVariables[EnvProcOOMScore]; ok {
+		score, err := strconv.Atoi(strings.TrimSpace(scoreStr))
+		if err == nil {
+			oomScore = score
```

Comment:
Should validate that it's within the expected range, if not log an error and fallback to default